package.json

```diff
@@ -21,9 +21,7 @@
     "url": "https://github.com/kaosat-dev/adafruit-i2c-pwm-driver/issues"
   },
   "dependencies": {
-    "underscore": "1.2.4",
-    "coffee-script": "1.3.3",
-    "i2c": "0.1.1",
+    "i2c": "0.2.3",
     "sleep": "~1.1.1"
```

Comment:
this didn't work for me on latest raspbian and `node@6.9.x` without `sleep@~3.0.0` .. that resolved to `sleep@3.0.1`